### PR TITLE
[Indigo] Fix bumper plugin

### PR DESCRIPTION
**Pull request by fmessmer**

### CODE REVIEW

File: cob_gazebo_worlds/urdf/ipa-apartment/elevator.urdf.xacro [hunk 1]
```diff
@@ -350,27 +350,25 @@
...
-  <!--gazebo reference="elevator_button1">
-    <selfCollide>true</selfCollide>
-        <sensor name='elevator_button1_contact_sensor' type='contact'>
-      <always_on>true</always_on>
+  <gazebo reference="elevator_button1">
+    <gravity>true</gravity>
+    <self_collide>false</self_collide>
+    <material>Gazebo/Red</material>
+    <sensor name='elevator_button1_contact_sensor' type='contact'>
+      <always_on>1</always_on>
```
Comment (floweisshardt) [inline cob_gazebo_worlds/urdf/ipa-apartment/elevator.urdf.xacro]: is "1" the same as "true"?

File: cob_gazebo_worlds/urdf/ipa-apartment/elevator.urdf.xacro [hunk 2]
```diff
@@ -408,24 +406,23 @@
...
-  <!--gazebo reference="elevator_button2">
-    <selfCollide>true</selfCollide>
-        <sensor name='elevator_button2_contact_sensor' type='contact'>
-      <always_on>true</always_on>
+  <gazebo reference="elevator_button2">
+    <gravity>true</gravity>
+    <self_collide>false</self_collide>
+    <material>Gazebo/Red</material>
+    <sensor name='elevator_button2_contact_sensor' type='contact'>
+      <always_on>1</always_on>
```
Comment (floweisshardt) [inline cob_gazebo_worlds/urdf/ipa-apartment/elevator.urdf.xacro]: is "1" the same as "true"?

**Comment by fmessmer** [comment]
using `true` again for consistency

**Comment by floweisshardt** [comment]
visual review: OK

**Pull request merged**